src/kixi/datastore/metadatastore.clj

```diff
@@ -52,7 +52,7 @@
   (s/cat :segmentations (s/+ ::segmentation)))
 
 (s/def ::filemetadata
-  (s/keys :req [::type ::id ::name ::schemastore/id ::provenance ::size-bytes]
+  (s/keys :req [::type ::id ::name ::schemastore/name ::provenance ::size-bytes]
```

Comment:
there isn't any. ::name here is the file name (really this is just a place holder for 'random file metadata'), schemastore/name is the name of the schema for this file.